npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
Avoid checking in log files. You can pick which files to check in by `git add {file_name|file pattern}`. Also check your diffs before committing by: `git diff`